File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/Meta.java
package work.socialhub.kmastodon.entity;

import com.google.gson.annotations.SerializedName;

/**
 * @author uakihir0
 */
public class Meta {

    @SerializedName("focus")
    private MetaFocus focus;
    @SerializedName("original")
    private MetaOriginal original;

    public MetaFocus getFocus() {
        return focus;
    }

    public void setFocus(MetaFocus focus) {
        this.focus = focus;
    }

    public MetaOriginal getOriginal() {
        return original;
    }

    public void setOriginal(MetaOriginal original) {
        this.original = original;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/Meta.kt
package work.socialhub.kmastodon.entity

import com.google.gson.annotations.SerializedName

/**
 * @author uakihir0
 */
class Meta {
    @SerializedName("focus")
    var focus: MetaFocus? = null

    @SerializedName("original")
    var original: MetaOriginal? = null
}
